.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on: push
+
+env:
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
+  DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
+  COMPOSE_FILE: docker-compose.test.yml
```

Comment:
Should we name this env variable as `DOCKER_COMPOSE_FILE` to keep the consistency with the other env variables?

Reply:
I got the reason, that env variable is used by docker-compose ([official documentation](https://docs.docker.com/compose/reference/envvars/#compose_file))
> Specify the path to a Compose file. If not provided, Compose looks for a file named docker-compose.yml in the current directory and then each parent directory in succession until a file by that name is found.
should we leave a comment above that env variable to avoid the confusion? or, if it's obvious and just I don't know about that fact, please let me know 🙈